contents/handbook/people/compensation.mdx

```diff
@@ -16,94 +16,102 @@ We think the fastest possible shipping comes from a leaner, stronger team, so we
 
 We will not adjust your pay downwards for existing team members if this ratio changes once you've already started working for PostHog. That means you can feel confident in your pay.
 
 ### Location factor
+
 ​
 We use GitLab's location factors to calculate what a fair market rate is for each location. GitLab uses a combination of data from Economic Research Institute (ERI), Numbeo, Comptryx, Radford, Robert Half, and Dice. [Read more](https://about.gitlab.com/handbook/total-rewards/compensation/compensation-calculator/#calculating-location-factors) on how GitLab calculates this location factor. If your location isn't listed, we will create one for you based on Numbeo's data for the relative Cost Of Living with San Francisco.
 ​
+
 ### Level
 
-More experience does not correlate with increased importance. Seniority is *not* a title - we don't believe in having a huge hierarchy of roles, as everyone needs to feel like the owner of the company that they are.
+More experience does not correlate with increased importance. Seniority is _not_ a title - we don't believe in having a huge hierarchy of roles, as everyone needs to feel like the owner of the company that they are.
 
 We pay more experienced team members a greater amount since it is reasonable to expect this correlates with an increase in skill - being able to ship faster through less time having to work things out for the first time is valuable. Experienced hires can help upskill the less experienced hires in the team too. Team members who have less experience can see a steady increase in pay over time as they increase their experience and skill.
 
-We believe at first increased skill comes from more time spent in the role. Over time, this judgement becomes more subjective and is instead based on the speed with which you can ship or help the team to ship, the quality of your prioritization and decision-making, as well as your technical approach. 
+We believe at first increased skill comes from more time spent in the role. Over time, this judgement becomes more subjective and is instead based on the speed with which you can ship or help the team to ship, the quality of your prioritization and decision-making, as well as your technical approach.
 
 ### Steps
 
 Within each level, we believe there's a place to have incremental steps. We define these as follows:
 
-- *Learning*: Starting to match expectations.
-- *Growing*: Matching expectations.
-- *Thriving*: Exceeding expectations.
-- *Expert*: Exceeding expectations consistently.
+-   _Learning_: Starting to match expectations.
+-   _Growing_: Matching expectations.
+-   _Thriving_: Exceeding expectations.
+-   _Expert_: Exceeding expectations consistently.
 
 The definition of what is needed to progress from one step to the next in more detail depends on your role. Ask your manager for detail of what you need to work on.
 
 ## Share Options
+
```

Comment:
why all the extra spaces? they don't appear to do anything?

Reply:
I'm not sure what has caused this. Maybe a linting/prettier thing? The only line changed was the one I highlighted.

Reply:
Your IDE maybe? Or did you run `prettier` manually? I ask because our pre-commit linting shouldn't affect .md files

Nevertheless, no problem at all! I'll get this in - hit me up at yakko [at] posthog [.] com for some merch!

Reply:
Nope, didn't run anything manually. I'll try to replicate the issue when I am on my other computer, I'll be sure to update you when I work it out. Thanks 🙂